CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 
 ### Removed Features
 - ROX-14398: As announced in 3.74, the permission `Access` replaces the deprecated permission `Role`.
+- ROX-14398: As announced in 3.74, the `Scope Manager` system role and permission set will be removed. If existing product instances do have customer references to either the system role or the system permission set, then the referenced object will be renamed and kept in its renamed form.
```

Comment:
just to make it a bit more clear on what it all entails, specifically mentioning that we do not support this role going forward seems worthwhile IMO:
```suggestion
- ROX-14398: As announced in 3.74, the `Scope Manager` system role and permission set will be removed. If existing RHACS instances do have references to either the `Scope Manager` role or the `Scope Manager` permission set, then the referenced object will be adjusted to contain a description mentioning it's deprecation. Furthermore, the object will not be marked as a system resource, and will not be supported going forward by RHACS.
```

Reply:
Recommend not to include `RHACS`/`stackrox` product name in changelog.

nit: `instances` -> `installations`